### PR TITLE
Pro 1052 add table endpoint documentation in SDK, Python, and Javascript

### PR DESCRIPTION
**Pull request by waddahAldrobi**
We are adding documentation to the `/create` and `/insert` table endpoints

### CODE REVIEW

File: api-reference/tables/endpoint/create.mdx [hunk 1]
```diff
@@ -20,14 +20,83 @@ The resulting table will be empty, and can be inserted into with the [/insert en
...
 
+```python Python SDK
+import dotenv, os
+from dune_client.client import DuneClient
+ 
+dotenv_path = os.path.join(os.path.dirname(__file__), '.', '.env')
+dotenv.load_dotenv(".env")
+dune = DuneClient.from_env()
+
+table = DuneClient.from_env().create_table(
```
Comment (hawkaa) [inline api-reference/tables/endpoint/create.mdx:43]:
```suggestion
table = dune.create_table(
```

File: api-reference/tables/endpoint/create.mdx [hunk 2]
```diff
@@ -20,14 +20,83 @@ The resulting table will be empty, and can be inserted into with the [/insert en
...
+        namespace="my_user",
+        table_name="dataset_interest_rates",
```
Comment (hawkaa) [inline api-reference/tables/endpoint/create.mdx:45]:
```suggestion
        table_name="interest_rates",
```

We removed this requirements and AFAIK our examples also removed this prefix.

File: api-reference/tables/endpoint/create.mdx [hunk 3]
```diff
@@ -20,14 +20,83 @@ The resulting table will be empty, and can be inserted into with the [/insert en
...
+```
+
+```python Python
+import requests
+
+url = "https://api.dune.com/api/v1/table/create"
+
+payload = {
+    "namespace": "my_user",
+    "table_name": "dataset_interest_rates",
```
Comment (hawkaa) [inline api-reference/tables/endpoint/create.mdx:62]:
```suggestion
    "table_name": "interest_rates",
```

File: api-reference/tables/endpoint/create.mdx [hunk 4]
```diff
@@ -20,14 +20,83 @@ The resulting table will be empty, and can be inserted into with the [/insert en
...
+    "description": "10 year daily interest rates, sourced from https://fred.stlouisfed.org/series/DGS10",
+    "schema": [{"name": "date", "type": "timestamp"}, {"name": "dgs10", "type": "double"}],
+    "is_private": false
+};
+
+const headers = {
+    "X-DUNE-API-KEY": "<x-dune-api-key>",
+    "Content-Type": "text/csv"
```
Comment (hawkaa) [inline api-reference/tables/endpoint/create.mdx:90]:
```suggestion
    "Content-Type": "application/json"
```

Comment (hawkaa) [reply api-reference/tables/endpoint/create.mdx:90]: The data you intend to upload might be a csv, but when you create the table the `payload` is a json object

File: api-reference/tables/endpoint/insert.mdx [hunk 5]
```diff
@@ -30,4 +30,61 @@ curl --request POST \
...
+
+```python Python SDK
+import dotenv, os
+from dune_client.client import DuneClient
+ 
+dotenv_path = os.path.join(os.path.dirname(__file__), '.', '.env')
+dotenv.load_dotenv(".env")
+dune = DuneClient.from_env()
+
+with open("./interest_rates.csv", "rb") as data:
```
Comment (hawkaa) [inline api-reference/tables/endpoint/insert.mdx:42]: Nice!!

File: api-reference/tables/endpoint/insert.mdx [hunk 6]
```diff
@@ -30,4 +30,61 @@ curl --request POST \
...
+
+```javascript Javascript
+const url = "https://api.dune.com/api/v1/table/my_user/dataset_interest_rates/insert";
+
+const headers = {
+    "X-DUNE-API-KEY": "<x-dune-api-key>",
+    "Content-Type": "text/csv"
+};
+
+fs.readFile('./interest_rates.csv', (err, data) => {
```
Comment (hawkaa) [inline api-reference/tables/endpoint/insert.mdx:74]: Did we test this? It looks correct, but haven't seen any examples of doing (async) file processing in javascript.

**Review by hawkaa — CHANGES_REQUESTED** [review]
Great work and thanks for doing this! I added some suggestions.

I see several other endpoints have snippets for Golang, PHP (?!) and Java. Should we add a task for that? Maybe we can include it in this PR?

@0xBoxer how do we test the examples? Or are we just crossing our fingers and hope they work?

**Comment by waddahAldrobi** [comment]
> I see several other endpoints have snippets for Golang, PHP (?!) and Java. Should we add a task for that? Maybe we can include it in this PR?

I noticed that too, but I wasn't sure if we should add them (from now), as if we iterate we will need to make changes across 7 examples instead of 4. However, I could add them, but I just followed what the Linear task listed.

> @0xBoxer how do we test the examples? Or are we just crossing our fingers and hope they work?

Do you mean test them in the web app? Or on our own? 
If the latter, then I have already tested every snippet :)

File: api-reference/tables/endpoint/create.mdx (hunk 2 above)
Comment (waddahAldrobi) [reply api-reference/tables/endpoint/create.mdx:45]: At the time of writing, without the `dataset_` was still not queryable :) 
And I was aligning with the `curl` command. 
But yes, I can change them to be without the prefix

File: api-reference/tables/endpoint/create.mdx (hunk 4 above)
Comment (waddahAldrobi) [reply api-reference/tables/endpoint/create.mdx:90]: Copy/paste typo 😄

File: api-reference/tables/endpoint/insert.mdx (hunk 6 above)
Comment (waddahAldrobi) [reply api-reference/tables/endpoint/insert.mdx:74]: Yes sir, it was tested. 
I'm not a javascript expert though, this was chatGPTed. 

Is there a different way you recommend doing this?

Comment (hawkaa) [reply api-reference/tables/endpoint/insert.mdx:74]: I have no idea! If you tested it that is fantastic!

**Comment by hawkaa** [comment]
> Do you mean test them in the web app? Or on our own? If the latter, then I have already tested every snippet :)

Meant the latter so we are good!

**Review by hawkaa — APPROVED** [review]

**Pull request merged**